spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -300,6 +301,11 @@ public Session getSession() {
 		 */
 		private final Accesslog accesslog = new Accesslog();
 
+		/**
+		 * Thread related configuration.
+		 */
+		private final Threads threads = new Threads();
```

Comment:
Looking at the code I am wondering now if `pool` wouldn't be better and more consistent with other properties of the same nature.

@bono007 no need to act on that now, I'd like to get some more feedback before asking you to change anything.

Reply:
Its interesting because both concepts are related but not exclusive of one another. Some servers may not have pool properties, some may have thread properties that are unrelated to pool, etc. 

I think `threads` is more generic than `pool` and most developers will associate that w/ the current thread related properties they are used to.

Reply:
After thinking about this some more, suppose I were to add another `maxQueueDepth` property to Jetty, this would not make sense under `threads` but would make sense under `pool`.